index.html

```diff
@@ -21,35 +21,39 @@ <h1 id="settings">Aperture Settings</h1>
 
   <p class="subheader">Improving your photography by controlling aperture</p>
 
-  <p>Cameras with interchangeable lenses offer three primary settings that affect how a photograph is exposed when the shutter button is pressed. These are aperture, shutter speed, and sensitivity (also called “ISO”) and are often referred to collectively as the “exposure triangle.” Each setting impacts the brightness of a photograph while also providing its own unique impact. This guide will focus on aperture.</p>
```

Comment:
I put the figure above the first paragraph so that everything flows neatly around it.